src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -101,6 +102,59 @@ We recommend tracking the following metrics:
 - `physical_replication.sst_bytes`: The [SST]({% link {{ page.version.version }}/architecture/storage-layer.md %}#ssts) bytes (compressed) sent to the KV layer by all physical cluster replication jobs.
 - `physical_replication.replicated_time_seconds`: The [replicated time]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}#cutover-and-promotion-process) of the physical replication stream in seconds since the Unix epoch.
 
+## Data verification
+
+{{site.data.alerts.callout_info}}
+**This feature is in [preview]({% link {{ page.version.version }}/cockroachdb-feature-availability.md %}).** It is in active development and subject to change.
+
+`SHOW EXPERIMENTAL_FINGERPRINT` allows you to verify that the data transmission and ingestion is working as expected while a replication stream is running. Any checksum mismatch likely represents corruption or a bug in CockroachDB. Should you encounter such a mismatch, contact [Support](https://support.cockroachlabs.com/hc/en-us).
+{{site.data.alerts.end}}
+
+To verify that the data up to a certain point in time is correct on the standby cluster, you can use the current replicated time from the replication job information to run a point-in-time fingerprint on both the primary and standby clusters. This will verify that the transmission and ingestion of the data on the standby cluster, up to that point in time, is correct.
+
+1. Retrieve the current replicated time of the replication job on the standby cluster with [`SHOW VIRTUAL CLUSTER`]({% link {{ page.version.version }}/show-virtual-cluster.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT replicated_time FROM [SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS];
+    ~~~
+    ~~~
+         replicated_time
+    ----------------------------
+    2023-12-15 19:39:18.321455+00
+    (1 row)
+    ~~~
+
+    For detail on connecting to the standby cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-standby-cluster-system-interface).
+
+1. From the **primary cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER application] AS OF SYSTEM TIME '2023-12-15 19:39:18.321455+00';
+    ~~~
+    ~~~
+    tenant_name |             end_ts             |     fingerprint
+    ------------+--------------------------------+----------------------
+    application | 1702669158089558000.0000000000 | 2646132238164576487
```

Comment:
Very minor: this output is likely from a different invocation of the command since the timestamps don't match.

If you'd like, use

`2024-01-09 16:15:45.291575+00` as the replicated time and then `1704816945291575000.0000000000` as the end_ts.

Reply:
Oh yes, helpful... I don't think I converted the original correctly!